spec/facets/ExtractionErrorRunFacet.json

```diff
@@ -1,6 +1,6 @@
 {
   "$schema": "https://json-schema.org/draft/2020-12/schema",
-  "$id": "https://openlineage.io/spec/facets/1-0-0/ExtractionErrorRunFacet.json",
+  "$id": "https://openlineage.io/spec/facets/1-0-1/ExtractionErrorRunFacet.json",
```

Comment:
Maybe this should be 1-1-0
even if this was an oversight initially, it does make this change not backward compatible.
https://github.com/OpenLineage/OpenLineage/blob/main/spec/Versioning.md#mechanism
and following the link there: https://docs.snowplow.io/docs/pipeline-components-and-applications/iglu/common-architecture/schemaver/
```
That's why we decided to introduce our own schema versioning notion: SchemaVer.
SchemaVer is defined as follows: MODEL-REVISION-ADDITION

MODEL when you make a breaking schema change which will prevent interaction with any historical data
REVISION when you introduce a schema change which may prevent interaction with some historical data
ADDITION when you make a schema change that is compatible with all historical data
```

Reply:
@julienledem  I fully do agree after reqading versioning guide. I'm changing both to 1-1-0

Reply:
Done.